### PR TITLE
registry: Added support for new region param

### PR DESCRIPTION
**Pull request by CollinShoop**
Not ready for merge until backend work is completed: CON-5476 & CON-5477

### CODE REVIEW

File: specification/resources/registry/get_registry_options.yml [hunk 1]
```diff
@@ -1,13 +1,19 @@
 operationId: get_registry_options
 
-summary: List Available Subscription Tiers
+summary: List Available Registry Options (Subscription Tiers and Available Regions)
```
Comment (waynr) [inline specification/resources/registry/get_registry_options.yml:3]: nit: s/List Available Registry Options/List Registry Options/

File: specification/resources/registry/models/registry_create.yml [hunk 2]
```diff
@@ -20,6 +20,18 @@ properties:
...
+    type: string
+    enum:
+    - nyc3
+    - sfo3
+    - ams3
+    - sgp1
+    - fra1
+    example: fra1
+    description: Slug of the region where registry data is stored. When not provided, a region will be selected based
+      on current load.
```
Comment (waynr) [inline specification/resources/registry/models/registry_create.yml:33]: nit: s/selected based on current load./selected./ (no need to mention anything about how the default is selected)

**Review by waynr — APPROVED** [review]
Just a couple nits, looks good otherwise!

**Review by ChiefMateStarbuck — APPROVED** [review]

**Comment by ChiefMateStarbuck** [comment]
!deploy

**Comment by CollinShoop** [comment]
@ChiefMateStarbuck  Thank you!!

**Pull request merged**